src/commands/receipt.ts

```diff
@@ -27,20 +27,21 @@ export const handler = async (argv: Arguments<Options>): Promise<void> => {
   const { hash, privateKey, chain, rpcRelay } = argv;
 
   try {
-    const signer = getWallet({
+    const signer = getSignerOnly({
```

Comment:
Now that polygon is the default, the previous hack of using getWallet here doesn't work. Because there is no default provider for polygon Mumbai built into ethersjs. But, since all we need here is a signer, we can use a different hack (that we had used previously). Both of these hacks are working around a regression in js-tableland that should actually be addressed directly.